statgen/pheget
# Minor ui updates

## Purpose
Minor UI updates
- Changed some wording and styling (made buttons smaller)
- Added navbars to region and single-variant views
- Moved style code to common.css

## How to test this
- Click navbar buttons and make sure they take you to the proper sections
- Make sure searchbox works correctly

## Deployment / configuration notes
(none)
